frontend/src/metabase/public/containers/PublicOrEmbeddedDashboard/PublicOrEmbeddedDashboardView-filters.stories.tsx

```diff
@@ -833,3 +892,85 @@ DarkThemeDateFilterRelative.play = async ({ canvasElement }) => {
     .getByRole("button", { name: "Yesterday" })
     .classList.add("pseudo-hover");
 };
+
+// Unit of time
+export const LightThemeUnitOfTime = Template.bind({});
+LightThemeUnitOfTime.args = createDefaultArgs({
+  parameterType: "temporal_unit",
+  parameterValues: {
+    [UNIT_OF_TIME_FILTER_ID]: "minute",
+  },
+});
+LightThemeUnitOfTime.play = async ({ canvasElement }) => {
+  const canvas = within(canvasElement);
+  const filter = await canvas.findByRole("button", {
+    name: "Unit of Time",
+  });
+  await userEvent.click(filter);
+
+  const popover = getLastPopover();
+  (popover.getByText("Hour").parentNode as HTMLElement).classList.add(
+    "pseudo-hover",
+  );
+};
+
+export const DarkThemeUnitOfTime = Template.bind({});
+DarkThemeUnitOfTime.args = createDefaultArgs({
+  theme: "night",
```

Comment:
Looks like `theme: night` is the only thing that differs between two stories. I wonder if you can create `UnitOfTimeStory` as a template, and bind `{theme: night}` and `{theme: dark}` so we don't repeat the play logic as well as parameter props?

Reply:
I could probably do that since most of the stories, light and dark are only different in the passed theme.

Reply:
Fixed in ad582ee5f168dd064690a177312aea9cd51ee266. I couldn't make it a template, since when we call `.bind()` we won't inherit any static properties.